descriptors/ModuleDescriptor-template.json

```diff
@@ -372,6 +372,7 @@
             "inventory-storage.location-units.institutions.item.get",
             "inventory-storage.location-units.campuses.item.get",
             "inventory-storage.location-units.libraries.item.get",
+            "inventory-storage.location-units.libraries.collection.get",
```

Comment:
I think we have been adding permissions/code dependencies but forgetting to add module dependencies as well. `location-units` is not in the `requires` list.

There could be others (I see `configuration` referenced without an explicit `requires` dependency), but other discrepancies should be handled by another JIRA/PR.

Reply:
Thanks for pointing that. Added `location-units` interface dependency.
Actually `configuration` dependency is present in the `requires` list.

Reply:
Hmm... I could have sworn `configuration` was not there. I must have been looking at a different MD. 😕